### ISSUE TITLE
assessNumberSignatures function

### ISSUE THREAD

**Issue by hongenxu** [julian-gehring/SomaticSignatures#6]
Hi, 

when I used

` gof_nmf = assessNumberSignatures(sca_mm, n_sigs, nReplicates = 5)`,

I got errors

Error in (function (x)  :
  unused arguments (model = list(model = "NMFstd", rank = 2, target = 0), method = "random")

Would you please help me？

my sessionInfo()

```
R version 3.4.0 (2017-04-21)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Fedora 20 (Heisenbug)

Matrix products: default
BLAS: /home/users/xu/local/lib64/R/lib/libRblas.so
LAPACK: /home/users/xu/local/lib64/R/lib/libRlapack.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C
 [9] LC_ADDRESS=C               LC_TELEPHONE=C
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C

attached base packages:
[1] stats4    parallel  stats     graphics  grDevices utils     datasets
[8] methods   base

other attached packages:
 [1] BSgenome.Hsapiens.1000genomes.hs37d5_0.99.1
 [2] BSgenome_1.44.0
 [3] rtracklayer_1.36.2
 [4] ggplot2_2.2.1
 [5] SomaticCancerAlterations_1.12.0
 [6] SomaticSignatures_2.12.0
 [7] VariantAnnotation_1.22.0
 [8] Rsamtools_1.28.0
 [9] Biostrings_2.44.0
[10] XVector_0.16.0
[11] SummarizedExperiment_1.6.1
[12] DelayedArray_0.2.4
[13] matrixStats_0.52.2
[14] Biobase_2.36.2
[15] GenomicRanges_1.28.2
[16] GenomeInfoDb_1.12.0
[17] IRanges_2.10.1
[18] S4Vectors_0.14.1
[19] BiocGenerics_0.22.0

loaded via a namespace (and not attached):
 [1] ProtGenerics_1.8.0            bitops_1.0-6
 [3] doParallel_1.0.10             RColorBrewer_1.1-2
 [5] httr_1.2.1                    tools_3.4.0
 [7] backports_1.0.5               R6_2.2.1
 [9] rpart_4.1-11                  Hmisc_4.0-3
[11] DBI_0.6-1                     lazyeval_0.2.0
[13] colorspace_1.3-2              nnet_7.3-12
[15] gridExtra_2.2.1               GGally_1.3.0
[17] compiler_3.4.0                graph_1.54.0
[19] htmlTable_1.9                 exomeCopy_1.22.0
[21] pkgmaker_0.22                 ggbio_1.24.0
[23] scales_0.4.1                  checkmate_1.8.2
[25] NMF_0.20.6                    proxy_0.4-17
[27] RBGL_1.52.0                   stringr_1.2.0
[29] digest_0.6.12                 foreign_0.8-68
[31] base64enc_0.1-3               dichromat_2.0-0
[33] htmltools_0.3.6               ensembldb_2.0.1
[35] htmlwidgets_0.8               rlang_0.1.1
[37] RSQLite_1.1-2                 BiocInstaller_1.26.0
[39] shiny_1.0.3                   BiocParallel_1.10.1
[41] acepack_1.4.1                 RCurl_1.95-4.8
[43] magrittr_1.5                  GenomeInfoDbData_0.99.0
[45] Formula_1.2-1                 Matrix_1.2-10
[47] Rcpp_0.12.11                  munsell_0.4.3
[49] stringi_1.1.5                 yaml_2.1.14
[51] zlibbioc_1.22.0               plyr_1.8.4
[53] AnnotationHub_2.8.1           grid_3.4.0
[55] lattice_0.20-35               splines_3.4.0
[57] GenomicFeatures_1.28.0        knitr_1.16
[59] rngtools_1.2.4                reshape2_1.4.2
[61] codetools_0.2-15              biomaRt_2.32.0
[63] XML_3.98-1.7                  biovizBase_1.24.0
[65] latticeExtra_0.6-28           pcaMethods_1.68.0
[67] data.table_1.10.4             httpuv_1.3.3
[69] foreach_1.4.3                 gtable_0.2.0
[71] reshape_0.8.6                 gridBase_0.4-7
[73] mime_0.5                      xtable_1.8-2
[75] AnnotationFilter_1.0.0        survival_2.41-3
[77] tibble_1.3.1                  OrganismDbi_1.18.0
[79] iterators_1.0.8               GenomicAlignments_1.12.1
[81] AnnotationDbi_1.38.0          registry_0.3
[83] memoise_1.1.0                 cluster_2.0.6
[85] interactiveDisplayBase_1.14.0
```

**Comment by juliangehring** [comment]
This error is related to upstream changes in the NMF package which is used for the non-negative matrix decomposition. Is is fixed in version 2.12.1 of the package - I have pushed the changes a few hours ago, and the package will become available through bioconductor within the next day (updated packages are built once a day, hence the slight delay). You can also check the current version here: https://bioconductor.org/packages/release/bioc/html/SomaticSignatures.html

**Comment by juliangehring** [comment]
Generally, please use the bioconductor support site for any issues or questions: https://support.bioconductor.org.
